src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -229,6 +229,12 @@
       "Whenever you apply #yBurning, apply that much #yBurning #b%1$s extra times."
     ]
   },
+  "stsjorbsmod:ReapAndSowPower": {
+    "NAME": "Reap and Sow",
+    "DESCRIPTIONS": [
+      "At the start of the next combat, attack ALL enemies for #b%1d damage."
```

Comment:
Probably best to avoid the term `attack` here, it might imply to the player that it should work like attack damage and apply vulnerable/weak/etc. I think it's probably better to implement it like you did and use non-attack damage (via the THORNS damageType), and update the wording to be more like `deal #b%1d damage to ALL enemies.`